### DISCUSSION TITLE
Why do the filters "Unplayed" and "Resumable" combine instead of adding? What is "resumable"?

### DISCUSSION THREAD

**Discussion by Elekam** [jellyfin/jellyfin#11379] [General]
When I select both "Unplayed" and "Resumable" in the filter for my series, I would think that both series I have not started, as well as series I have watched a few episodes of, or even stopped mid-episode, would be shown.

That is not the case. When I select both, nothing is shown. In general, when I select "Resumable" nothing shows up.
I have two questions:
1. What even is "resumable"?
 I assumed it would be either series I have started but not finished, or series in which I stopped in the middle of an episode. I definitely have both right now, so why is nothing shown?
2. Why would the filters combine, like the series has to be both "resumable" and "unplayed"?
Is that not impossible? Unless I seriously misunderstand what "resumable" means, it doesn't make any sense. How can something be both resumable and unplayed?

Thanks in advance for any answers! :)

![image](https://github.com/jellyfin/jellyfin/assets/43169119/3ace0ba9-72e2-46b7-a05d-81a2cba0255d)

**Comment by nielsvanvelzen** [comment]
Resumable means partially played items (so movies/episodes). Unplayed means that it contains one or multiple items that have not been played (for series/seasons/playlists/collections etc.)

**Comment by Elekam** [comment]
Then resumable seems to not work in my Jellyfish instance for some reason. When I only tick resumable, nothing appears. But I see on my homepage like 8 different series under "Continue watching" that are in the middle of an episode, so how can nothing show up?
